packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/Providers/ModelFactoryProvider.cs

```diff
@@ -345,7 +345,8 @@ private static IReadOnlyList<ValueExpression> GetCtorArgs(
                 {
                     if (IsNonReadOnlyMemoryList(factoryParam))
                     {
-                        expressions.Add(factoryParam.NullConditional().ToList());
+                        // non-ReadOnlyMemory lists are already initialized to a non-null value
```

Comment:
Does this imply that ROM lists still need the null conditional or are they handled differently?

Reply:
They are handled differently. I can remove this comment as it is confusing.